mobile/lib/src/application/device/device_selector_cubit.dart

```diff
@@ -30,13 +30,26 @@ class DeviceSelectorCubit extends Cubit<DeviceState> {
   Future<void> connect(AcquisitionDevice device) async {
     emit(DeviceConnectionInProgress());
 
+<<<<<<< HEAD
```

Comment:
conflict

Reply:
I noticed, I'm fixing them right now. It's weird they weren't showing up in my vscode, clean open everything.